src/components/cc-tile-scalability/cc-tile-scalability.types.d.ts

```diff
@@ -1,52 +1,7 @@
-export interface Consumption {
-  yesterday: number,
-  last30Days: number,
+interface Scalability {
+  minFlavor: Flavor;
+  maxFlavor: Flavor;
```

Comment:
`Flavor` does not exist in this file. Should we import it from common or move all this to common ?